gaia-flex/big-dipper-settings.json

```diff
@@ -0,0 +1,54 @@
+{
+    "public":{
+        "chainName": "Gaia Flex",
+        "chainId": "gaia-flex",
+        "gtm": "{Add your Google Tag Manager ID here}",
+        "slashingWindow": 10000,
+        "uptimeWindow": 250,
+        "initialPageSize": 30,
+        "bech32PrefixAccAddr": "cosmos",
+        "bech32PrefixAccPub": "cosmospub",
+        "bech32PrefixValAddr": "cosmosvaloper",
+        "bech32PrefixValPub": "cosmosvaloperpub",
+        "bech32PrefixConsAddr": "cosmosvalcons",
+        "bech32PrefixConsPub": "cosmosvalconspub",
+        "bondDenom": "uatom",
```

Comment:
Wait... this is `umuon`

Reply:
I got these values from [Default big-dipper](https://github.com/forbole/big-dipper/blob/master/default_settings.json) settings assuming these are the values that are used in cosmos mainnet. Are you sure?